sfge/src/main/java/com/salesforce/rules/RuleUtil.java

```diff
@@ -65,6 +65,8 @@ public static List<MethodVertex> getPathEntryPoints(
             methods.addAll(MethodUtil.getRemoteActionMethods(g, fileLevelTargets));
             // ...and PageReference methods...
             methods.addAll(MethodUtil.getPageReferenceMethods(g, fileLevelTargets));
+            // ...and global-exposed methods...
```

Comment:
Do we need a RuleUtilTest for this category as well?

Reply:
Can't hurt. Stand by for that.